bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/seqcomp/GhostIExpSeqComponent.java

```diff
@@ -0,0 +1,12 @@
+package edu.gemini.spModel.seqcomp;
+
+public interface GhostIExpSeqComponent extends IObserveSeqComponent {
```

Comment:
A random musing that you should ignore: the `I` stuff in interface names feels a bit antiquated. Beyond that though, should the exposure times appear here since it won't be used for standard observes or bias?  Do you need one that just has exposure counts and then an extension with exposure times?

Reply:
Yes, it look like I will be needing to split this into two.

Reply:
I split this into three interfaces:
* `GhostExpTimeSeqComponent`: exposure times
* `GhostExpCountSeqComponent`: exposure counts
* `GhostExpSeqComponent`: subclass of both (thus exposure times and counts).